src/util/Constants.js

```diff
@@ -709,6 +711,21 @@ exports.WebhookTypes = [
   'Channel Follower',
 ];
 
+/**
+ * The value set for a sticker's type:
+ * * PNG
+ * * APNG
+ * * LOTTIE
+ * @typedef {string} StickerFormatTypes
+ */
+exports.StickerFormatTypes = [
```

Comment:
use the new `createEnum` function here maybe ?

this export also needs to be added to the typings

Reply:
I have added the missing type.

Should I use createEnum like `createEnum([null, 'PNG', 'APNG', 'LOTTIE'])` ?

Reply:
yes

Reply:
I have committed could you review it?